mobile/status.go

```diff
@@ -84,9 +84,13 @@ func initializeApplication(requestJSON string) string {
 	providers.MixpanelAppID = request.MixpanelAppID
 	providers.MixpanelToken = request.MixpanelToken
 
-	datadir := request.DataDir
+	mediaServerEnableTLS := true
+	if request.MediaServerEnableTLS != nil {
+		mediaServerEnableTLS = *request.MediaServerEnableTLS
+	}
+	statusBackend.StatusNode().UpdateMediaServerEnableTLS(mediaServerEnableTLS)
```

Comment:
I guess it's `true` by default anyway?
```suggestion
	if request.MediaServerEnableTLS != nil {
		statusBackend.StatusNode().UpdateMediaServerEnableTLS(*request.MediaServerEnableTLS)
	}
```